horace_core/sqw/coord_transform/@ortho_proj/private/init_by_input_parameters_.m

```diff
@@ -0,0 +1,50 @@
+function obj = init_by_input_parameters_(obj,varargin)
+%INIT_BY_INPUT_PARAMETERS_ is the helper allowing to support various mainly
+%outdate forms of input of the ortho_proj constructor.
+%
+% Inputs:
+% list of input parameters of projection, produced as set of positional
+% parameters and then key-value, key-value pairs.
+%
+% The positional parameters order is:
+%
+% u,v,w,nonorthogonal,type,alatt,angdeg,offset,label,title,lab1,lab2,lab3,lab4
+% First non-positional parameter considered to be key.
```

Comment:
It's not clear how to use this. What are the key parameters? Or can the positional parameters also be key parameters. Some more explanation is needed

Reply:
this is standard usage of serializable. You can not say the same thing for each of its instances.